server/models/migrations/v0.8.0-v0.9.0/migrate.sql

```diff
@@ -1044,3 +1044,23 @@ INSERT INTO `account_reference_type` (`id`, `label`, `fixed`) VALUES
 -- Account Reference Type unit
 INSERT INTO unit VALUES
 (225, 'Account Reference Type','TREE.ACCOUNT_REFERENCE_TYPE','Account Reference Type', 1,'/modules/account_reference_type','/account_reference_type');
+
+
+DROP TABLE IF EXISTS `debtor_group_history`;
+CREATE TABLE `debtor_group_history` (
+  `uuid` BINARY(16) NOT NULL,
+  `debtor_uuid` BINARY(16) DEFAULT NULL,
+  `previous_debtor_group` BINARY(16) DEFAULT NULL,
+  `next_debtor_group` BINARY(16) DEFAULT NULL,
+  `income_date` timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP,
```

Comment:
I know you were not the original author, but what is `income_date`?  Can we just make it called `created_at`, like most timestamps?